code/modules/surgery/organs/organ_external.dm

```diff
@@ -638,6 +633,23 @@ Note that amputating the affected organ does in fact remove the infection from t
 		owner.handle_splints()
 	return TRUE
 
+/obj/item/organ/external/proc/internal_bleed()
+	if(is_robotic())
+		return
+	if(HAS_TRAIT(owner, NO_BLOOD))
+		return
+	status |= ORGAN_INT_BLEEDING
+	owner.custom_pain("You feel something rip in your [name]!")
+
+/obj/item/organ/external/proc/fix_internal_bleeding()
+	if(is_robotic())
+		return FALSE
+	if(!(status & ORGAN_INT_BLEEDING))
+		return FALSE
```

Comment:
I'd say remove this. We don't need to know if they have this flag or not beforehand. If you try to remove a flag from a bitfield which does not have that flag, nothing will happen.
```suggestion
```